modules/src/what_cocktail.py

```diff
@@ -0,0 +1,351 @@
+#rewrite the program (which a burglar stole from me!) and then release it onto telegram and messenger!
+''' argh what cocktail can i make with these ingredients? search and return a cocktail which has
+the most matching ingredients'''
+from enchant.checker import SpellChecker
+import re
+
+print("What? Cocktail")
+print()
+print()
+
+def search(drinks, dictionary):
+    filter_dict = {}
+    for key, value in dictionary.items():
+        for v in value:
+            if v == drinks:
+                filter_dict.update({key: value})
+    drinks = filter_dict
+    return drinks
+
+
+def spell_check(word):
+    if dict_lookup(word, cocktails) is True:
+        return word
+
+    elif word == "Rum":
+        word = input("White or Dark?").title()
+        word += " Rum"
+        chkr = SpellChecker("en_GB", word)
+        for err in chkr:
+            while dict_lookup(word, cocktails) is not True:
+                print("Your spelling wasn't recognised")
+                print()
+                print("Are you trying to spell one of these?")
+                print()
+                suggestion = str(err.suggest(word))
+                suggestion = re.sub(r'(\')', '', suggestion)
+                suggestion = re.sub(r'\[', '', suggestion)
+                suggestion = re.sub(r'\]', '', suggestion)
+                print(suggestion)
+                print()
+                word = input("Please re-type it:").title()
+                print()
+                if word == "White":
+                    word = "White Rum"
```

Comment:
return word instead of assigning it.

Reply:
sorry not sure what you mean here. I do return it but I also assign it. What would you like me to change?

Reply:
You don't use the variable `word` so why bother assigning it? Instead do `return "White Rum"` or `return "Dark Rum"` instead. Or assign it to `word` and have just one return at the end of the function